pkg/kdump/kdump.scss

```diff
@@ -19,9 +19,9 @@
 
 @use "page";
 
-@import "global-variables";
-@import "@patternfly/patternfly/utilities/Text/text.scss";
-@import "@patternfly/patternfly/utilities/Spacing/spacing.scss";
+@use "global-variables" as *;
+@use "@patternfly/patternfly/utilities/Text/text.scss";
+@use "@patternfly/patternfly/utilities/Spacing/spacing.scss";
```

Comment:
Is moving `@import` to `@use` and `@forward` strictly related to font changes? It's not obvious, and the commit message doesn't explain it. If it's a prerequisite, could you split out the import changes into a separate PR with an explanation, and we land that first (better regression checking). That should not cause pixel changes.

A separate commit in this PR also works for me, as you prefer.